src/zenml/zen_server/deploy/helm/templates/serviceaccount.yaml

```diff
@@ -9,4 +9,8 @@ metadata:
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
+{{- with .Values.imagePullSecrets }}
+imagePullSecrets:
```

Comment:
The image pull secrets should be defined already in the Deployment's PodSpec so that it works for ZenServer. However, adding this to the service account is also a valid thing to do, in this case. Did you have any specific use-case in mind?

Reply:
When using a private regitry this allows access to it for all the pipelines without  needing to define it for every pipeline you write. It was a recommendation of @francoisserra on Slack